test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -637,6 +641,23 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 					return nil
 				}, 2*time.Minute, 5*time.Second, "checking scheduling domains with pod running")
 			})
+
+			It("[test_id:73382]cpuset.cpus.exclusive of kubepods.slice should be updated", func() {
+				if !cgroupV2 {
+					Skip("cpuset.cpus.exclusive is part of cgroupv2 interfaces")
+				}
+				cpusetCfg := &controller.CpuSet{}
+				err := getter.Container(ctx, testpod, testpod.Spec.Containers[0].Name, cpusetCfg)
+				Expect(err).ToNot(HaveOccurred())
+				podCpuset, err := cpuset.Parse(cpusetCfg.Cpus)
+				kubepodsExclusiveCpus := fmt.Sprintf("%s/kubepods.slice/cpuset.cpus.exclusive", cgroupRoot)
+				cmd := []string{"cat", kubepodsExclusiveCpus}
+				exclusiveCpus, err := nodes.ExecCommandOnNode(ctx, cmd, workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+				exclusiveCpuset, err := cpuset.Parse(exclusiveCpus)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(podCpuset).To(Equal(exclusiveCpuset))
```

Comment:
Let's not compare strings here.
We should use `podCpuset.Equal(exclusiveCpuset)` because it compares the actual cpu values.
An example for why not using strings:
setA = `1,2,3`
setB = `1-3`
sets are logically equals but strings are different.

Reply:
podCpuset and exclusiveCpuset are type cpuset.CPUset